client/js/commands/search.js

```diff
@@ -2,17 +2,28 @@
 
 import store from "../store";
 import {router} from "../router";
+const Msg = require("../../../src/models/msg");
```

Comment:
Should we really include server/backend code in the front end? This will be the first case of importing anything from `/src/` inside the `/client/`
CC/ @MaxLeiter

Reply:
Yeah... I need input there. I didn't want to make a fake message and just opted in for using the message model, but yes, that'd indeed be the first case.

Reply:
Yeah, we shouldn't import msg. We don't currently have a nice way of generating client-side messages, so we'd need to pass it to the server. Regardless, when thinking about accessibility, a chat message isn't the best way to notify the user of an error (at least, with our current set-up). With that in mind, maybe we should use an alert or modal.